pipeline-runner/R/gem2s-1-download_cellranger.R

```diff
@@ -47,13 +54,11 @@ download_cellranger <- function(input, pipeline_config) {
     config$metadata <- input$metadata
   }
 
-  exportJSON <- RJSONIO::toJSON(config)
-
-  message("META file (config):")
-  message(exportJSON)
-
-  write(exportJSON, "/input/meta.json")
+  prev_out$config <- config
+  res <- list(
+    data = list(),
+    ouput = prev_out)
 
-  message("Step 1 complete")
-  return(list())
+  message("\nStep 1 complete")
```

Comment:
Wouldn't it be more helpful if you said "Download of cellranger files step complete"? What if in the future we introduced a step before that one, then this will no longer be step 1.